schema/spacewalk/upgrade/susemanager-schema-4.3.12-to-susemanager-schema-4.3.13/100-rocky-epel.sql

```diff
@@ -0,0 +1,11 @@
+-- Rocky Linux 9
```

Comment:
Please move the script to `susemanager-schema-4.4.0-to-susemanager-schema-4.4.1/100-rocky-epel.sql`.

I think the folder does not exist yet, so you can create it.